Mathlib/Topology/ContinuousMap/CompactlySupported.lean

```diff
@@ -541,6 +540,104 @@ noncomputable def nnrealPart (f : C_c(α, ℝ)) : C_c(α, ℝ≥0) where
 lemma nnrealPart_apply (f : C_c(α, ℝ)) (x : α) :
     f.nnrealPart x = Real.toNNReal (f x) := rfl
 
+instance : Coe C(α, ℝ≥0) C(α, ℝ) where
+  coe := fun f => ⟨fun x => NNReal.toReal (f.toFun x), by continuity⟩
```

Comment:
This strikes me as a perhaps surprising coercion. Could you at least write it using `ContinuousLinearMap.comp`?

Reply:
Could you elaborate why this is surprising? I thought it were natural because there is a coercion `(X → ℝ≥0) → (X → ℝ)`. Yet most probably this should be in another file

Reply:
> there is a coercion `(X → ℝ≥0) → (X → ℝ)`

Wait, where did you get that from? The following doesn't work:
```
import Mathlib

open scoped NNReal

example {X : Type*} (f : X → ℝ≥0) : X → ℝ := f
```

Reply:
mmm sorry, I thought I had that coercion (and I thought I had done a test twice on Lean 4 web!) but you are right, it does not work. So should I not define the coercion?

Reply:
Yes, I think that coercion is a bad idea. Better write things explicitly in terms of `ContinuousLinearMap.comp`

Reply:
I see, I removed the coercions